packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -27,31 +23,22 @@ const GuidanceWrapper = styled.div`
 `;
 
 const Content = styled.div`
-  display: flex;
-  align-items: center;
-  padding: ${GEL_SPACING_DBL};
+  padding: ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL};
+  }
```

Comment:
Is `Content` needed now that we don't have nested content, like the icon? You could safely move these styles into `GuidanceWrapper` and prune it from this component.

Reply:
Thanks @simonsinclair, removed!